<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1275</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(165, 165, 167);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>List of people</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>List of books</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Borrowed books</string>
    </property>
   </widget>
   <widget class="QComboBox" name="people_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>271</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="books_list">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>271</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="borrow_button">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Borrow the book</string>
    </property>
   </widget>
   <widget class="QComboBox" name="books_return">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>520</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="return_book_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>550</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Return the book</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Add a new person</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_name">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>30</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Surname</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_surname">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>80</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_person">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>110</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add the person</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>-10</y>
      <width>16</width>
      <height>591</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>581</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>10</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove selected person from list</string>
    </property>
   </widget>
   <widget class="QComboBox" name="remove_person">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>30</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_button">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>70</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove person from list</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>150</y>
      <width>681</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>0</y>
      <width>20</width>
      <height>361</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>170</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Add a new book</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_title">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>190</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_author_name">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>230</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_author_surname">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>270</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>190</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Book title</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>230</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Author's name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>270</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Author's surname</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>170</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove selected book from list</string>
    </property>
   </widget>
   <widget class="QComboBox" name="remove_book">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>190</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_book_button">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>230</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove book from list</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>350</y>
      <width>681</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="add_book_button">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>310</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add the book</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>470</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>You can only remove a book that is not borrowed by someone.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>440</y>
      <width>511</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>You can only remove a person from list who doesn't have borrowed books currently.</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="people">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>271</width>
      <height>191</height>
     </rect>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="books">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>271</width>
      <height>191</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="books_borrow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>551</width>
      <height>181</height>
     </rect>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1275</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
